bin/develop

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+trap "kill 0" EXIT
```

Comment:
again, im a big lover of comments ... quickly outline what running this accomplishes?

Reply:
Hmm, I picked that up from this post: https://spin.atomicobject.com/2017/08/24/start-stop-bash-background-process/

But it doesn't seem to matter whether I have it or not? I thought it was necessary to kill child processes, but maybe I'm misunderstanding